src/Serilog/Settings/KeyValuePairs/SettingValueConversions.cs

```diff
@@ -29,6 +35,22 @@ class SettingValueConversions
 
         public static object ConvertToType(string value, Type toType)
         {
+            if (TryParseStaticMemberAccessor(value, out var accessorTypeName, out var memberName))
+            {
+                var accessorType = Type.GetType(accessorTypeName, throwOnError: true);
```

Comment:
Probably, for a better UX could we use full type names without specifying assembly? We could look for the target type in configuration assemblies (`using` section) something like:
```csharp
var accessorType = Type.GetType(typeName) ??
            _assemblies.SelectMany(a => a.ExportedTypes)
                        .FirstOrDefault(t => t.FullName == typeName);
```

Reply:
If we do that for members, could/should we do the same thing for instances?

Reply:
For now I've tried to not deviate too much from the way arbitrary sub-types (for interfaces and abstract classes) are referenced, that is with full assembly-qualified type names. Maybe lifting that restriction would be nice, but I would keep it as a separate issue / PR.

Reply:
Sounds good